src/extension/features/accounts/hide-memos/components/hide-memos-button.jsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { controllerLookup } from 'toolkit/extension/utils/ember';
+import { l10n, getToolkitStorageKey } from 'toolkit/extension/utils/toolkit';
+
+export const HideMemosButton = ({ toggleHiddenState }) => {
+  const notHidden = getToolkitStorageKey('hide-memos', true);
+  const label = notHidden ? l10n('app.show', 'is-checked') : l10n('app.hide', 'not-checked');
+
+  const toggleHidden = () => {
+    toggleHiddenState(!notHidden);
+    controllerLookup('application').send('closeModal');
+    document.getElementById('ember1434').remove();
+  };
+
+  return (
+    <div
+      title="Hide Memo Column"
+      id="ember1434"
+      className="modal-account-view-menu ynab-checkbox ember-view"
+    >
+      <button
+        onClick={toggleHidden}
+        className={`ynab-checkbox-button js-ynab-checkbox-button ${label}`}
```

Comment:
I know ynab has these classes on their element that you're mimicking but they don't seem to be doing anything, we can just remove them (same thing with `ember-view` above)

Reply:
I removed all the classes that didn't seem to break anything.  modal-account-view-menu is required to keep the text from turning bold.